### PR TITLE
Nginx setup

### PR DESCRIPTION
**Pull request by emma-p**
- Instead of using the conventional `setup.sh` script for nginx, specifies manual instructions. Reasoning:
  - every user installing should be aware of how the nginx setup works, especially since some operations require `sudo`
  - the script is hard to debug when it fails and can leave the machine in an unexpected state
  - the number of steps to perform is relatively small

- Uses port 8852 instead of default 9000 (risk of conflicts with other applications)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,52 @@
...
+* Make sure you are in the base `identity-admin` directory
+
+* Make sure you have access to the S3 bucket identity-local-ssl and download them using the [AWS CLI utility](https://aws.amazon.com/cli/) (the following command will download them in your current directory using your Identity profile on AWS):
+
+```bash
+aws --profile identity s3 cp s3://identity-local-ssl/idadmin-thegulocal-com-exp2018-09-16-bundle.crt . 1>/dev/null
+aws --profile identity s3 cp s3://identity-local-ssl/idadmin-thegulocal-com-exp2018-09-16.key . 1>/dev/null
+```
+
+* Find the configuration folder of nginx on your machine:
```
Comment (susiecoleman) [inline README.md]: I'd change this to "Find the configuration folder of nginx on your machine using:"

File: README.md [hunk 2]
```diff
@@ -1 +1,52 @@
...
+
+```bash
+nginxHome=`nginx -V 2>&1 | grep "configure arguments:" | sed 's/[^*]*conf-path=\([^ ]*\)\/nginx\.conf.*/\1/g'`
+```
+
```
Comment (susiecoleman) [inline README.md]: I think an example of a test using "echo $nginxHome" would make it a bit clearer what the pervious step did

**Comment by susiecoleman** [comment]
Looks good, I've suggested a couple of minor rewordings for clarity but apart from that :+1:

**Comment by nlindblad** [comment]
Nice :+1:

**Pull request merged**